compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -54,17 +54,17 @@ class ElimRepeated extends MiniPhase with InfoTransformer { thisPhase =>
                       |""".stripMargin,
               sym.sourcePos)
           else
-            addVarArgsForwarder(sym, isJavaOverride, hasAnnotation)
+            addVarArgsForwarder(sym, isJavaVarargsOverride, hasAnnotation)
       else if hasAnnotation
         report.error("A method without repeated parameters cannot be annotated with @varargs", sym.sourcePos)
     end
 
     super.transform(ref) match
-      case ref1: SymDenotation if ref1.is(Method) =>
+      case ref1: SymDenotation if ref1.is(Method, butNot=JavaDefined) =>
```

Comment:
Spaces around the = sign:
```suggestion
      case ref1: SymDenotation if ref1.is(Method, butNot = JavaDefined) =>
```